sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -71,6 +77,45 @@ export function createRecordedCommunicationIdentityClient(
   };
 }
 
+export function createRecordedCommunicationIdentityClientWithToken(
+  context: Context
+): RecordedClient<CommunicationIdentityClient> {
+  const recorder = record(context, environmentSetup);
+
+  if (isBrowser()) {
+    return {
+      client: new CommunicationIdentityClient(env.COMMUNICATION_CONNECTION_STRING),
+      recorder
+    };
+  }
+
+  let credential: TokenCredential;
+
+  if (isPlaybackMode()) {
+    const mockToken = "token";
+    const fakeGetToken = sinon.fake.returns(
```

Comment:
Can we avoid adding sinon just for this?

Reply:
removed the sinon lib